src/CommunityToolkit.Maui/Essentials/FileFolderDialog.tizen.cs

```diff
@@ -0,0 +1,390 @@
+﻿using System.ComponentModel;
+using Microsoft.Maui.Controls.Compatibility.Platform.Tizen;
+using Microsoft.Maui.Platform;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.UIExtensions.NUI;
+using Path = System.IO.Path;
+using Button = Tizen.UIExtensions.NUI.Button;
+using Entry = Tizen.UIExtensions.NUI.Entry;
+using Label = Tizen.UIExtensions.NUI.Label;
+using TColor = Tizen.UIExtensions.Common.Color;
+using ScrollView = Tizen.UIExtensions.NUI.ScrollView;
+using Shadow = Tizen.NUI.Shadow;
+using HorizontalAlignment = Tizen.NUI.HorizontalAlignment;
+using VerticalAlignment = Tizen.NUI.VerticalAlignment;
+using View = Tizen.NUI.BaseComponents.View;
+using Window = Tizen.NUI.Window;
+
+namespace CommunityToolkit.Maui.Storage;
+
+enum FileSelectionMode
```

Comment:
This looks like to be the same enum from Android, can this be at `shared` lvl?

Reply:
Android implementation is reworked from scratch. Now this enum is only used in this file

Reply:
Oooh! You could maybe use the new `file` scope, introduced in C# 11 https://learn.microsoft.com/dotnet/csharp/language-reference/proposals/csharp-11.0/file-local-types